app/controllers/base.rb

```diff
@@ -88,6 +88,11 @@ def blank_is_nil!(hash)
 					end
 				end
 			end
+
+			def render_error(status_code, message)
```

Comment:
do we want to use this, or use exceptions we catch and render a standard error template?